### PR TITLE
Add URL links in-line

### PR DESCRIPTION
**Pull request by Sing-Li**

### CODE REVIEW

File: client/src/components/Home/index.js [hunk 1]
```diff
@@ -249,11 +249,11 @@ export default function Home(props) {
               <p>
                 <a href="#" className="header-link">
```
Comment (RonLek) [inline client/src/components/Home/index.js]: We'll need to replace the `#` here by the `https://docs.rocket.chat/guides/user-guides` (with the quotes) for it to redirect.

File: client/src/components/Home/index.js [hunk 2]
```diff
@@ -249,11 +249,11 @@ export default function Home(props) {
...
                   {t("unsigned-home-demo.user-guides")}
-                </a>{" "}
+                </a>{"https://docs.rocket.chat/guides/user-guides"}
```
Comment (RonLek) [inline client/src/components/Home/index.js]: The `{"https://docs.rocket.chat/guides/user-guides"}` after the `a` tag is currently rendering as a text. (Can be seen [here](https://i.postimg.cc/zBShRFYc/Screenshot-from-2021-03-24-19-50-42.png)). We need to remove it.

File: client/src/components/Home/index.js [hunk 3]
```diff
@@ -249,11 +249,11 @@ export default function Home(props) {
...
                 |{" "}
                 <a href="#" className="header-link">
```
Comment (RonLek) [inline client/src/components/Home/index.js]: Same for this one. We need to replace the `#` by `https://github.com/RocketChat/Rocket.Chat/releases`.

File: client/src/components/Home/index.js [hunk 4]
```diff
@@ -249,11 +249,11 @@ export default function Home(props) {
...
                   {t("unsigned-home-demo.release-notes")}
-                </a>
+                </a>{"https://github.com/RocketChat/Rocket.Chat/releases"}
```
Comment (RonLek) [inline client/src/components/Home/index.js]: The `{"https://github.com/RocketChat/Rocket.Chat/releases"}` part should be removed here. (It renders as text currently)

**Review by RonLek — CHANGES_REQUESTED** [review]

File: client/src/components/Home/index.js (hunk 1 above)
Comment (Sing-Li) [reply client/src/components/Home/index.js]: done thx

File: client/src/components/Home/index.js (hunk 2 above)
Comment (Sing-Li) [reply client/src/components/Home/index.js]: ok

File: client/src/components/Home/index.js (hunk 4 above)
Comment (Sing-Li) [reply client/src/components/Home/index.js]: ok

File: client/src/components/Home/index.js [hunk 5]
```diff
@@ -247,11 +247,11 @@ export default function Home(props) {
               <br />
 
               <p>
-                <a href="#" className="header-link">
+                <a href="https://docs.rocket.chat/guides/user-guides" className="header-link">
                   {t("unsigned-home-demo.user-guides")}
-                </a>{" "}
```
Comment (RonLek) [inline client/src/components/Home/index.js:252]: This causes a minor spacing issue here. I'll cover it up in the next one. We can go ahead!

**Review by RonLek — APPROVED** [review]
LGTM! @Sing-Li . We can go ahead!

**Pull request merged**